content/authorization/reference/architecture/register/_index.nb.md

```
@@ -1,22 +1,121 @@
-Registerkomponenten er en .NET-applikasjon som eksponerer API. Foreløpig fungerer den som en proxy mot Altinn 2-registeret,
-men vi arbeider med å legge til lokal lagring for bedre ytelse og for å kunne fase ut Altinn 2 i 2026.
+ER-registeret kan brukes til å forhåndsutfylle informasjon, hente informasjon basert på brukerinput eller for å validere brukerinput.
 
-Planlagt design:
+I løpet av høsten 2025 vil registerkomponenten bli oppdatert for å få sin egen
 
-![Register design](registerdesign.drawio.svg)
+Registerkomponenten er en ASP.Net Core MVC-applikasjon som eksponerer REST-API til Altinn Apps.
 
-## Database design
+Løsningen er nå tilgjengelig på https://platform.altinn.cloud/register/api/v1 og alle ressurser er tilgjengelige gjennom endepunktene definert nedenfor.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)